spec/spec_helper.rb

```diff
@@ -5,4 +5,127 @@
   SimpleCov::Formatter::HTMLFormatter,
   Coveralls::SimpleCov::Formatter
 ]
-Coveralls.wear!
+Coveralls.wear! 
+
+# Generated by rspec-sinatra. (2015-11-07 23:21:32 +0300)
+
+ENV['RACK_ENV'] = 'test'
+
+require File.join(File.dirname(__FILE__), '..', 'app/chitter.rb')
+
+require 'capybara'
+require 'capybara/rspec'
+require 'rspec'
+require 'database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.